### PR TITLE
sync exercise docs

### PR DESCRIPTION
**Pull request by glaxxie**
Changes are 100% auto generated from the problem specifications

### CODE REVIEW

File: exercises/practice/anagram/.meta/tests.toml [hunk 1]
```diff
@@ -46,6 +46,11 @@ description = "detects anagrams using case-insensitive possible matches"
 
 [7cc195ad-e3c7-44ee-9fd2-d3c344806a2c]
 description = "does not detect an anagram if the original word is repeated"
+include = false
+
+[630abb71-a94e-4715-8395-179ec1df9f91]
```
Comment (IsaacG) [inline exercises/practice/anagram/.meta/tests.toml]: Is this supposed to be matched with a test change?

File: exercises/practice/custom-set/.meta/tests.toml [hunk 2]
```diff
@@ -75,6 +75,9 @@ description = "Sets with the same elements are equal -> sets with different elem
 [06059caf-9bf4-425e-aaff-88966cb3ea14]
 description = "Sets with the same elements are equal -> set is not equal to larger set with same elements"
 
+[d4a1142f-09aa-4df9-8b83-4437dcf7ec24]
+description = "Sets with the same elements are equal -> set is equal to a set constructed from an array with duplicates"
```
Comment (IsaacG) [inline exercises/practice/custom-set/.meta/tests.toml]: Should this be matched with a test change?

File: exercises/practice/dnd-character/.meta/tests.toml [hunk 3]
```diff
@@ -65,3 +65,8 @@ description = "random character is valid"
 
 [2ca77b9b-c099-46c3-a02c-0d0f68ffa0fe]
 description = "each ability is only calculated once"
+include = false
+
+[dca2b2ec-f729-4551-84b9-078876bb4808]
```
Comment (IsaacG) [inline exercises/practice/dnd-character/.meta/tests.toml]: Should this be matched with a test change?

File: exercises/practice/grains/.meta/tests.toml [hunk 4]
```diff
@@ -27,11 +27,17 @@ description = "returns the number of grains on the square -> grains on square 16
...
 [61974483-eeb2-465e-be54-ca5dde366453]
-description = "returns the number of grains on the square -> negative square raises an exception"
+description = "returns the number of grains on the square -> negative square is invalid"
 
 [a95e4374-f32c-45a7-a10d-ffec475c012f]
-description = "returns the number of grains on the square -> square greater than 64 raises an exception"
+description = "returns the number of grains on the square -> square greater than 64 is invalid"
+
+[6eb07385-3659-4b45-a6be-9dc474222750]
+description = "returns the total number of grains on the board"
```
Comment (IsaacG) [inline exercises/practice/grains/.meta/tests.toml]: Ditto

File: exercises/practice/isbn-verifier/.meta/tests.toml [hunk 5]
```diff
@@ -9,6 +9,9 @@
 # As user-added comments (using the # character) will be removed when this file
 # is regenerated, comments can be added via a `comment` key.
 
+[0caa3eac-d2e3-4c29-8df8-b188bc8c9292]
+description = "valid isbn"
```
Comment (IsaacG) [inline exercises/practice/isbn-verifier/.meta/tests.toml]: Ditto

**Review by IsaacG — CHANGES_REQUESTED** [review]
The docs changes all look fine. The meta test changes all look suspect.

**Review by IsaacG — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**